src/hooks/use-focus-trap.ts

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```

Comment:
Can we reuse here [focusTrap utils](https://github.com/gravity-ui/uikit/blob/76394b464ad67479482754fc06f1279e2cbaa989/src/components/utils/FocusTrap.tsx) from uikit?  
Do we really need it? `Popup` now contains [some focus trap](https://github.com/gravity-ui/uikit/blob/76394b464ad67479482754fc06f1279e2cbaa989/src/components/Popup/Popup.tsx#L7)  logic

Reply:
Probably not, I'll check